packages/playground/src/app/Header/SettingsSubmenu.tsx

```diff
@@ -0,0 +1,42 @@
+import { InputSwitch } from '@paritytech/components/';
+import { ReactElement, useContext } from 'react';
+import { AppContext } from '~/context';
+import { Dispatch, State } from '~/context/reducer';
+
+export const SettingsSubmenu = (): ReactElement => {
+  const [state, dispatch]: [State, Dispatch] = useContext(AppContext);
+
+  return (
+    <div className="mr-8 w-44">
+      <div className="py-2 flex justify-between">
```

Comment:
Have you thought of a component "LabeledToggle" or does PrimeReact maybe provide something like this?

Reply:
No, not yet, could be a good idea, not sure if it adds unnecessary overhead. But I am open to that.

Reply:
LabeledInputSwitch component is implemented.